vulnfeeds/vulns/vulns.go

```diff
@@ -364,3 +365,19 @@ func (v *Vulnerability) ToYAML(w io.Writer) error {
 	encoder := yaml.NewEncoder(w)
 	return encoder.Encode(v)
 }
+
+func FromJSON(r io.Reader) (*Vulnerability, error) {
+	decoder := json.NewDecoder(r)
+	var vuln Vulnerability
+	err := decoder.Decode(&vuln)
+	if err != nil {
+		return nil, err
+	}
+
+	return &vuln, nil
+}
+
+func (v *Vulnerability) ToJSON(w io.Writer) error {
+	encoder := json.NewEncoder(w)
```

Comment:
Can you add 
```
encoder.SetIndent("", "  ")
```
it makes the output much easier to read, which helps to check if output is correct.

Reply:
Can we at least make this optional? formatting can be easily done by e.g. using `'cat blah.json | jq`.

Reply:
Sure, done.

Reply:
Okay, conflicting requests. @another-rex are you happy to just post-process it for human readability purposes? If so, I'll back [e9a140e](https://github.com/google/osv.dev/pull/748/commits/e9a140ec9515d7e75894fcdd4e871ff31edd24cf) out.

Reply:
Sure, let's just post process it.